app/controllers/public_controller.rb

```diff
@@ -1,12 +1,24 @@
 class PublicController < ActionController::Base
 
   protect_from_forgery with: :exception
+  include ApplicationHelper
 
-  def landing_page
+  def about
   end
 
   def community_resources
+    @admin_status = params[:admin] ? YAML.load(params[:admin]) : current_user&.admin?
```

Comment:
this was commented in another PR, and once we add Pundit should be reviewed

Reply:
@exbinary i wanted a way in dev to be able to view as non-admin, and expecting this to be removed once we add in Pundit (but would be open to allowing a params[:admin] == false to accept a downgrade!)